apps/web/src/app/(dashboard)/documents/data-table-action-dropdown.tsx

```diff
@@ -114,7 +114,7 @@ export const DataTableActionDropdown = ({ row, team }: DataTableActionDropdownPr
       <DropdownMenuContent className="w-52" align="start" forceMount>
         <DropdownMenuLabel>Action</DropdownMenuLabel>
 
-        {recipient?.role !== RecipientRole.CC && (
+        {recipient && recipient?.role !== RecipientRole.CC && (
```

Comment:
🤔

Reply:
Prior to this change, this will render a blank area if the user owns the document but is none of the recipients